saturday.tex

```diff
@@ -51,10 +51,10 @@ \section*{Saturday Sessions Descriptions}\label{saturday-descriptions}
 }{%
   Karen M. Sandler is the executive director of the Software Freedom Conservancy, which is the
   nonprofit home of dozens of free and open source software projects including Git, Etherpad, QEMU,
```

Comment:
I think nonprofit should be hyphenated?

Reply:
I've added #18 so that it is dealt with in a future pull request